src/components/DayPicker.jsx

```diff
@@ -365,14 +369,19 @@ export default class DayPicker extends React.Component {
   }
 
   onPrevMonthClick(nextFocusedDate, e) {
+    const { translationValueRTL } = this.state;
+    const { isRTL } = this.props;
+
     if (e) e.preventDefault();
 
     if (this.props.onPrevMonthClick) {
       this.props.onPrevMonthClick(e);
     }
 
-    const translationValue =
-      this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
+    let translationValue = this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
```

Comment:
you know what forget the rule disabling. i'll just add a new line (i've seen you guys doing it before i think, so i guess that's legit by you)